tests/end-to-end/upgrade/pkg/tests/eventmesh-migration/helpers/subscriber.go

```diff
@@ -0,0 +1,115 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/avast/retry-go"
+	appsv1 "k8s.io/api/apps/v1"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+)
+
+// TODO(k15r): change image to a release image, not PR-XXX image
```

Comment:
Oops

Reply:
I would like to keep it as is since it came from backup and we gotta solve this everywhere once and for all.